lib/kitchen/directions/bake_chapter_answer_key/v1.rb

```diff
@@ -17,7 +19,10 @@ def bake(chapter:, metadata_source:, strategy:, append_to:)
             <h2 data-type="document-title">
               <span class="os-text">#{I18n.t(:chapter)} #{chapter.count_in(:book)}</span>
             </h2>
-            #{metadata_source.copy}
+            <div data-type="metadata" style="display: none;">
+              <h1 data-type="document-title" itemprop="name">#{I18n.t(:chapter)} #{chapter.count_in(:book)}</h1>
+              #{metadata_source.children_to_keep.copy.paste}
```

Comment:
Just as a note, copying & pasting is what adds the `_copy_XXX` suffix (with `XXX` replaced by generated numbers). This allows for us to make sure that all ids are unique.